app/_data/alumni_india.yml

```diff
@@ -30,6 +30,75 @@
     LinkedIn: https://www.linkedin.com/in/aryanraj/
     Twitter: https://twitter.com/arynrj
 
+- name: Anish Mukherjee
+  pic: anish.jpeg
+  location: Goldman Sachs
+  position: Software Developer
+  links: 
+    LinkedIn: https://www.linkedin.com/in/alphadose/
+    Twitter: https://twitter.com/alphadose_
+
+- name: Ankit Kataria
+  pic: kataria.jpeg
+  location: Cisco
+  position: SDE
+  links: 
+    LinkedIn: https://www.linkedin.com/in/ankitkataria28/
+    Twitter: https://twitter.com/ankitkataria28
+
+- name: Kartikey Pandey
+  pic: kp.jpeg
+  location: Microsoft
+  position: Software Engineer
+  links: 
+    LinkedIn: https://www.linkedin.com/in/pandeykartikey/
+    Twitter: https://twitter.com/pandeykartikey_
+
+- name: Kriti Singh
+  pic: kriti.jpg
+  location: Goldman Sachs
+  position: Analyst
+  links: 
+    LinkedIn: https://www.linkedin.com/in/kriti-singh-9b5bb512b/
+    Twitter: https://twitter.com/kritisingh1ks
+
+- name: Nikhil kaushik
+  pic: nikhil.jpeg
+  location: Cohesity
+  position: Research Engineer
+  links: 
+    LinkedIn: https://www.linkedin.com/in/kriti-singh-9b5bb512b/
```

Comment:
umm.. are you sure that's my linkedin profile? :)

Reply:
@c0dzilla Sorry, I'll just make the changes! :joy: